dt/DTs_Designate/DFG-network-openstack-designate-17.1_director-rhel-virthost-3cont_2comp-ipv6-geneve.md

```diff
@@ -0,0 +1,64 @@
+# Deployed Topology 3_ocp_workers_2_compute_ipv6_geneve
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change              |  Date   |
+| -------: | :------------------ | :-----: |
+|     v0.1 | Initial publication | 2023-12-12 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Networker nodes                            | vm    | 3      |
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+|-----------|------------------------------| ------------------ |
+| Neutron   | ML2/OVN, Geneve              | Must have          |
+| Glance    | Ceph/RBD                     | Interchangable     |
+| Cinder    | Ceph/RBD Backend             | Interchangable     |
+| Keystone  | Special appliance [redacted] | Interchangable     |
+| Compute   | set-nova-scheduler-filter    | Must have          |
+| Designate | Geneve                       | Must have          |
+### Additional information 
+
+#### Nova
+
+#### Cinder
+
+#### OVN
+add configuration of ovn extras, ipv6
```

Comment:
what does "ovn extras" mean here? If it was anything related to tripleo and infrared, it needs to be redefined properly here.
Also, is this about IPv6 only or also IPv6?